helper/include/odp/helper/cli.h

```diff
@@ -54,20 +59,55 @@ typedef struct {
  */
 void odph_cli_param_init(odph_cli_param_t *param);
 
+/**
+ * Initialize CLI helper
+ *
+ * This function initializes the CLI helper. It must be called before
+ * odph_cli_register_command() and odph_cli_start().
+ *
+ * @param instance ODP instance
+ * @param param CLI server parameters to use
+ * @retval 0 Success
+ * @retval <0 Failure
+ */
+int odph_cli_init(const odp_instance_t instance, const odph_cli_param_t *param);
```

Comment:
To me "const" is not necessary for instance, as application cannot modify its content anyway.

Similarly, init_local() does not have const:
int odp_init_local(odp_instance_t instance, odp_thread_type_t thr_type)